source/getting-started/connection.txt

```diff
@@ -0,0 +1,622 @@
+================
+Connection Guide
+================
+
+.. default-domain:: mongodb
+
+Overview
+--------
+
+This guide shows you how to connect to MongoDB with the Node.js driver using
```

Comment:
1) Whoops, good catch. Will fix.

2) Will fix to make it more clear that they are common options. I actually mentioned to Nathan this morning that I was concerned the guide contained at least two different contexts: auth mechanisms and connection options, and that maybe it would be better to split them. I could even picture each auth mechanism on its own page since it's not common that a reader will need the context from more than one mechanism.